api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -350,7 +353,9 @@ public WorkspaceResponse getWorkspace(String projectName, String workspaceName)
 
   @Override
   public List<WorkspaceResponse> getWorkspaces() {
```

Comment:
I'd prefer to have this take the field list as a parameter. Right now this specifies a seemingly arbitrary (looking just at this class) subset of fields which in reality needs to be the supserset of all fields which are used currently throughout the application. At a conceptual level, the service shouldn't know this. Tying this instead to the callsite which knows what fields are needed for that context would be much more explicit.

Reply:
That seems reasonable to me. I was mirroring what we did for getWorkspace, but since there's only one callsite for this, it makes it lot easier to avoid duplicated code.